libs/ui-lib/lib/ocm/components/clusterConfiguration/review/ReviewNetworkingTable.tsx

```diff
@@ -91,9 +101,24 @@ export const ReviewNetworkingTable = ({ cluster }: { cluster: Cluster }) => {
         cells: [
           { title: 'Ingress IP' },
           {
-            title: selectIngressVip(cluster),
-            props: { 'data-testid': 'ingress-vip', colSpan: 2 },
+            title: cluster.ingressVips?.map((ingressVip) => (
+              <span key={ingressVip.ip}>
+                {ingressVip.ip}
+                <br />
+              </span>
+            )),
+            props: { 'data-testid': 'api-vip' },
           },
+          isDualStack({ ...cluster, openshiftVersion: cluster.openshiftVersion })
```

Comment:
why are you reconstructing cluster object with the same value for `openshiftVersion` ?